public/data/dunyanong.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Dun Yan",
+    "type": "personal",
+    "bio": "I am a self-taught developer who focuses on web developent and data sciences",
+    "avatar": "https://github.com/dunyanong.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/dunyanong",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/dun_yan_",
+        "icon": "twitter"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```